### PR TITLE
fix local build CMD

### PR DESCRIPTION
**Pull request by philipp-baumann**
Hi, thanks for making this great package! While testing local build, I noticed I had to adapt the command for the local build. Hence this mini PR. Cheers, Philipp

### CODE REVIEW

**Comment by philipp-baumann** [comment]
The other option would be to do:

```sh
cd ..
R CMD INSTALL --no-multiarch --with-keep.source r-polars
```

**Comment by sorhawell** [comment]
@philipp-baumann  thx alot for spotting this <3

**Comment by philipp-baumann** [comment]
> @philipp-baumann  thx alot for spotting this <3
> 
> Seem you're right on my local computer and as far as I can see :)
> 
> Maybe we should start recommending the Makefile.

ok cool, no prob.

**Comment by philipp-baumann** [comment]
@sorhawell like the Make approach you suggested.

**Review by sorhawell — APPROVED** [review]

File: README.Rmd [hunk 1]
```diff
@@ -248,7 +248,7 @@ cd r-polars
 **Step 4 (optional):** Build the package locally.
 
   ```r
```
Comment (eitsupi) [inline README.Rmd:250]:
```suggestion
  ```sh
```

**Comment by philipp-baumann** [comment]
thank you @eitsupi

File: README.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ devtools::test()     # run all unit tests
 **Step 4 (optional):** Build the package locally.
 
 ``` r
```
Comment (eitsupi) [inline README.md:308]: Could you update README.md too?

Comment (philipp-baumann) [reply README.md]: > Could you update README.md too?

sure, sorry for forgetting.

**Comment by philipp-baumann** [comment]
@eitsupi just noticed the same `r` instead of `sh` a little bit before in the `README.*`: 

- Option A: Using **devtools**.

  ``` r
  Rscript -e 'devtools::install(pkg = ".", dependencies = TRUE)' 
  ```

=> do you want a separate PR for this or shall I squash into this PR?

**Comment by eitsupi** [comment]
@philipp-baumann Thanks for taking a look at that. I'm glad you made the changes in this PR.

**Review by eitsupi — APPROVED** [review]
Thanks!

**Pull request merged**